### ISSUE TITLE
RemoteTransferManager log spamming

### ISSUE THREAD

**Issue by calestyo** [dCache/dcache#5327]
Hi.

Also in 6.0.9, the trasfermanager spams the log with:
```
Feb 25 03:08:20 lcg-lrz-srm dcache@transfermanagers[20489]: 2020-02-25 03:08:20+01:00 (RemoteTransferManager) [door:webdav.tls_lcg-lrz-dc13@webdav_lcg-lrz-dc13:AAWfXPY1ucg webdav.tls_lcg-lrz-dc13 RemoteTransferManager] stopTimer(): timer not found for Id=1582596500087000
Feb 25 03:08:20 lcg-lrz-srm dcache@transfermanagers[20489]: 2020-02-25 03:08:20+01:00 (RemoteTransferManager) [door:webdav.tls_lcg-lrz-dc13@webdav_lcg-lrz-dc13:AAWfXPY/8vg webdav.tls_lcg-lrz-dc13 RemoteTransferManager] stopTimer(): timer not found for Id=1582596500756000
Feb 25 03:08:21 lcg-lrz-srm dcache@transfermanagers[20489]: 2020-02-25 03:08:21+01:00 (RemoteTransferManager) [door:webdav.tls_lcg-lrz-dc14@webdav_lcg-lrz-dc14:AAWfXPZKpUA webdav.tls_lcg-lrz-dc14 RemoteTransferManager] stopTimer(): timer not found for Id=1582596501457000
Feb 25 03:08:21 lcg-lrz-srm dcache@transfermanagers[20489]: 2020-02-25 03:08:21+01:00 (RemoteTransferManager) [door:webdav.tls_lcg-lrz-dc14@webdav_lcg-lrz-dc14:AAWfXPZJl7g webdav.tls_lcg-lrz-dc14 RemoteTransferManager] stopTimer(): timer not found for Id=1582596501942000
Feb 25 03:08:48 lcg-lrz-srm dcache@transfermanagers[20489]: 2020-02-25 03:08:48+01:00 (RemoteTransferManager) [door:webdav.tls_lcg-lrz-dc14@webdav_lcg-lrz-dc14:AAWfXPfkCfA webdav.tls_lcg-lrz-dc14 RemoteTransferManager] stopTimer(): timer not found for Id=1582596528979000
Feb 25 03:08:51 lcg-lrz-srm dcache@transfermanagers[20489]: 2020-02-25 03:08:51+01:00 (RemoteTransferManager) [door:webdav.tls_lcg-lrz-dc14@webdav_lcg-lrz-dc14:AAWfXPgNvbg webdav.tls_lcg-lrz-dc14 RemoteTransferManager] stopTimer(): timer not found for Id=1582596531021000
Feb 25 03:08:52 lcg-lrz-srm dcache@transfermanagers[20489]: 2020-02-25 03:08:52+01:00 (RemoteTransferManager) [door:webdav.tls_lcg-lrz-dc14@webdav_lcg-lrz-dc14:AAWfXPgjtrg webdav.tls_lcg-lrz-dc14 RemoteTransferManager] stopTimer(): timer not found for Id=1582596532461000
Feb 25 03:08:53 lcg-lrz-srm dcache@transfermanagers[20489]: 2020-02-25 03:08:53+01:00 (RemoteTransferManager) [door:webdav.tls_lcg-lrz-dc14@webdav_lcg-lrz-dc14:AAWfXPgkvHA webdav.tls_lcg-lrz-dc14 RemoteTransferManager] stopTimer(): timer not found for Id=1582596533150000
Feb 25 03:08:59 lcg-lrz-srm dcache@transfermanagers[20489]: 2020-02-25 03:08:59+01:00 (RemoteTransferManager) [door:webdav.tls_lcg-lrz-dc14@webdav_lcg-lrz-dc14:AAWfXPiTXMA webdav.tls_lcg-lrz-dc14 RemoteTransferManager] stopTimer(): timer not found for Id=1582596539777000
Feb 25 03:09:01 lcg-lrz-srm dcache@transfermanagers[20489]: 2020-02-25 03:09:01+01:00 (RemoteTransferManager) [door:webdav.tls_lcg-lrz-dc14@webdav_lcg-lrz-dc14:AAWfXPileXg webdav.tls_lcg-lrz-dc14 RemoteTransferManager] stopTimer(): timer not found for Id=1582596541568000
```

As you can see... every few seconds...

Any idea what these messages mean?

Thanks,
Chris.

**Comment by paulmillar** [comment]
This problem is fixed in master, but wasn't back-ported to 6.0 because it wasn't seen as a high priority.  The message, although annoying, should be a rare occurrence on a well-running machine.  If the message shows up many times then something is wrong.

The problem is triggered when the client initiating an HTTP-TPC cancels that TPC-request before a TPC mover is started.

From 24a1c04's commit message
```
    The problem arises because, when the door cancels a transfer, it does
    not know whether or not the mover has started.  If the mover has started
    then there will be a timer task in transfermanager that kills the
    transfer if it takes too long.  If present, that timer must be
    cancelled.
    
    Therefore, if the mover hasn't started when the door cancels a transfer
    the transfermanager will attempt to cancel a timer that does not exist,
    triggering an error.
```

Ultimately, this points to HTTP-TPC requests failing, most likely because the request was queued  on a pool for longer than the client (FTS) timeout.

**Comment by calestyo** [comment]
> This problem is fixed in master, but wasn't back-ported to 6.0 because it wasn't seen as a high priority. 

Wasn't aware of that... and can absolutely live with it...

So from my side you can close the issue.

Thanks,
Chris.

**Comment by paulmillar** [comment]
OK, thanks.
